### PR TITLE
fixed initial bugs for mobile and did some work with images

### PR DESCRIPTION
**Pull request by sjmog**
Fixed some image display on mobile devices within media components within sliders.

Specifically, re-sized the image so it was well sexier and also re-sized the blockquote.

### CODE REVIEW

**Comment by sjmog** [comment]
Closes #68

**Comment by sjmog** [comment]
@pitchinvasion HEY
YEAH YOU

File: source/visual_guide/templates/slider.erb [hunk 1]
```diff
@@ -14,73 +14,72 @@
...
-          link: "#"
-        }
-      ]
-    }
-  ]
-} %>
+	<%= partial :"templates/slider", locals: {
+		type: "holler_blocks",
+		content: [
+			{
```
Comment (pitchinvasion) [inline source/visual_guide/templates/slider.erb]: Hmm I'm dubious is the tabbing here correct?

**Comment by pitchinvasion** [comment]
:+1:

File: source/visual_guide/templates/slider.erb (hunk 1 above)
Comment (sjmog) [inline source/visual_guide/templates/slider.erb]: Sure why not

**Pull request merged**